### DISCUSSION TITLE
CSV Export and Import

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#6462] [Help]
Our business guys are used to use Excel and want to do some batch changes in their "known environement". 

The question is:
is it possible to prepare a data table from a set of mongodb collections, where i need to lookup other data by id.
Then i provide an export button to get the data as CSV and import them back when data is changed.
I have seen an Export Action to CSV which is great, but is there also an import possible?

**Comment by melohagan** [comment]
Hi @buerobuettner 

You can import CSV data into a data table within the builder:
![Screenshot 2022-06-27 at 14 56 47](https://user-images.githubusercontent.com/101575380/175958924-13c03d19-9f4e-4658-adf0-a6c96fcfd6a7.png)
 
However there is no **Import CSV** action.

**Comment by ghost** [comment]
I cannot donwload CSV because i use mongodb as datasource (i guess) and i cannot upload and sync..
so i need to find another solution out there.

Thx anyway

**Reply by melohagan** [reply]
At the moment the MongoDB support is fairly basic. You can perform CRUD operations through custom queries, but the integration with Budibase tables isn't as complete as it is for external SQL databases, e.g. MySQL, PostgreSQL and MS SQL Server.
